[notifications] update Send notifications with FCM and APNs docs (#35420)
expo/expo: docs/pages/push-notifications/sending-notifications-custom.mdx

}
```

> The `experienceId` and `scopeKey` fields are only applicable to usage with Expo Go (prior to SDK 53, when push notifications support was removed from Expo Go). Otherwise, your notifications will not go through to your app. FCM has a list of supported fields in the [notification payload](https://firebase.google.com/docs/cloud-messaging/http-server-ref#notification-payload-support), and you can see which ones are supported by `expo-notifications` on Android by looking at the [FirebaseRemoteMessage](/versions/latest/sdk/notifications/#firebaseremotemessage).

Reviewer: ```suggestion
The `experienceId` and `scopeKey` fields are only applicable when using Expo Go (from SDK 53, push notifications support is removed from Expo Go). Otherwise, your notifications will not go through to your app. FCM has a list of supported fields in the [notification payload](https://firebase.google.com/docs/cloud-messaging/http-server-ref#notification-payload-support), and you can see which ones are supported by `expo-notifications` on Android by looking at the [FirebaseRemoteMessage](/versions/latest/sdk/notifications/#firebaseremotemessage).
```

I could be wrong, and please feel free to correct me, I think SDK 53 is the first SDK where the support is removed from Expo Go? If it is, let's we can rephrase the message above to improve readability.

Author: yes, what you're saying is correct. TY! :)

Reviewer: Thank you for clarifying this in the docs and for me :)
